pipeline_ukbiobank/cli/compute_stats.py

```diff
@@ -31,14 +30,32 @@
 hdlr = logging.StreamHandler(sys.stdout)
 logging.root.addHandler(hdlr)
 
-PREDICTORS = ['Sex', 'Height', 'Weight', 'Age', 'Vscale', 'Volume ventricular CSF', 'Brain GM volume', 'Brain WM volume', 'Total brain volume norm', 'Total brain volume', 'Volume of thalamus (L)', 'Volume of thalamus (R)'] # TODO Add units of each
+# Dictionnary of the predictors with units
+PREDICTORS = {
+    'Sex': '',
+    'Height': 'cm',
+    'Weight': 'kg',
+    'Age': 'y.o.',
+    'Vscale': '',
+    'Volume ventricular CSF': 'mm\u00b2',
```

Comment:
weird characters-- maybe replace with "^"?

Reply:
the goal was to display the 2 as an exponent like this:
![image](https://user-images.githubusercontent.com/71230552/125972584-5068f6b5-6a8d-4983-8f77-6d684e904ad9.png)

But in the saved .txt file it still displays as ^2, so i'll change it back!